rio_tiler/io/cogeo.py

```diff
@@ -95,8 +95,11 @@ class COGReader(BaseReader):
     # Define global options to be forwarded to functions reading the data (e.g rio_tiler.reader._read)
     nodata: Optional[Union[float, int, str]] = attr.ib(default=None)
     unscale: Optional[bool] = attr.ib(default=None)
-    vrt_options: Optional[Dict] = attr.ib(default=None)
     resampling_method: Optional[Resampling] = attr.ib(default=None)
+    vrt_options: Optional[Dict] = attr.ib(default=None)
+    post_process: Optional[
+        Callable[[numpy.ndarray, numpy.ndarray], Tuple[numpy.ndarray, numpy.ndarray]]
```

Comment:
Is
```py
Callable[[numpy.ndarray, numpy.ndarray], Tuple[numpy.ndarray, numpy.ndarray]]
```

fine instead of
```py
Callable[Tuple[numpy.ndarray, numpy.ndarray], Tuple[numpy.ndarray, numpy.ndarray]]
```
?

Reply:
@kylebarron if we do that, then the postprocess callable has be be 
```python
data, mask = post_process((data, mask))
```
instead of 
```python
data, mask = post_process(data, mask)
```

I'm not a big fan of passing a tuple as input but if you think it makes more sense or is more user friendly 👌

Reply:
Oh I see, no I agree I was just confused on the typings